src/middlewared/middlewared/plugins/pool.py

```diff
@@ -3383,6 +3388,13 @@ async def do_update(self, id, data):
                 if data['volsize'] < dataset[0]['volsize']['parsed']:
                     verrors.add('pool_dataset_update.volsize',
                                 'You cannot shrink a zvol from GUI, this may lead to data loss.')
+                else:
+                    # means the zvol size has increased so we need to check if
```

Comment:
It looks like we are just performing checks now and nothing is resized yet, or am I missing something?

Reply:
You're right, I'll fix